### DISCUSSION TITLE
Management of connected devices

### DISCUSSION THREAD

**Discussion by Didr0** [RaspAP/raspap-webgui#1128] [Q&A] [answered]
Hi, I heve installed raspap on my raspberry pi 3 B and I found this project awesome but... After some test I've found two little problems:
-1 I can't manage devices that are already connected (for example i can't disconnect them);
-2 I can't manage different networks adaptor (for example i have the wlan0 interfaces which I want to use to share 5 GHz network and wlan1 interface that i can't use in meanwhile for share 2.4 GHz network). 
I hope that someone can help me for this.
Thanks in advance...

**Comment by billz** [comment, marked as answer]
RaspAP uses `ip -o link show` to enumerate all currently available interfaces—physical, virtual, etc. If you disconnect a device (for example, by detaching a USB modem or Wi-Fi adapter) this will be reflected in the UI. The Insiders Edition has additional support for [managing network devices](https://docs.raspap.com/net-devices/) but this does not currently include disconnecting them. 

Support for multiple APs (in your example, using separate 5 GHz and 2.4 GHz adapters) would require substantial changes to RaspAP's internals. However, a workaround is proposed here https://docs.raspap.com/multiple/

PR's are welcome if you'd like to add support for either of these features.
